src/sentry/static/sentry/app/views/settings/components/forms/fieldSeparator.jsx

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+export default function FieldSeparator(props) {
```

Comment:
Would be nice to see this in a separate PR.

Reply:
Noted. Next time (and especially when not dealing with the settings UI). I need to get this out and don't want to start the rebase game anymore.